### PR TITLE
Mention that additional Go profile types are available

### PR DESCRIPTION
**Pull request by felixge**
### What does this PR do?

Mention that additional Go profile types are available.

### Motivation

The fact that not all profile types are enabled by default was not documented, neither was how to enable the additional ones.

### Preview

I'm part of the Datadog organization, but I'm new and not sure how to submit my PR in a way that will generate a preview yet.

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by sarina-dd — APPROVED** [review]

File: content/en/tracing/profiler/getting_started.md [hunk 1]
```diff
@@ -185,29 +185,33 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 
 **Note**:
 
+- By default only the CPU and Heap profile are enabled. Use [profiler.WithProfileTypes][4] to enable additional [profile types][5].
+
 - It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use profiler configuration to set the parameters:
```
Comment (dbenamydd) [inline content/en/tracing/profiler/getting_started.md:190]: I know you didn't add this but while we're here... :-)

```suggestion
- You can set profiler parameters in code with these functions:
```

File: content/en/tracing/profiler/getting_started.md [hunk 2]
```diff
@@ -185,29 +185,33 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
...
 | Method | Type          | Description                                                                                                  |
 | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
-|  WithService     | String        | The Datadog [service][4] name, for example `my-web-app`.             |
-|  WithEnv         | String        | The Datadog [environment][5] name, for example, `production`.         |
+|  WithService     | String        | The Datadog [service][6] name, for example `my-web-app`.             |
+|  WithEnv         | String        | The Datadog [environment][7] name, for example, `production`.         |
 |  WithVersion     | String        | The version of your application.                                                                             |
 |  WithTags        | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`. |
 
 - Alternatively you can also set the profiler configuration using environment variables:
```
Comment (dbenamydd) [inline content/en/tracing/profiler/getting_started.md:199]:
```suggestion
- Alternatively you can set profiler configuration using environment variables:
```

File: content/en/tracing/profiler/getting_started.md [hunk 3]
```diff
@@ -185,29 +185,33 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 
 **Note**:
 
+- By default only the CPU and Heap profile are enabled. Use [profiler.WithProfileTypes][4] to enable additional [profile types][5].
+
 - It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use profiler configuration to set the parameters:
 
 | Method | Type          | Description                                                                                                  |
```
Comment (dbenamydd) [inline content/en/tracing/profiler/getting_started.md:192]:
```suggestion
| Function | Type          | Description                                                                                                  |
```

**Review by dbenamydd — COMMENTED** [review]
Thanks!

**Pull request merged**